From 7aef7fb4c17670560005509021dd38fb96ba8f0f Mon Sep 17 00:00:00 2001
From: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
Date: Sun, 17 Feb 2013 11:11:30 +0100
Subject: [PATCH][BUILD] Fix systemd service creation

Need to specify path for the resulting file otherwise it fails.

Signed-off-by: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
---
 data/Makefile.am |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/Makefile.am b/data/Makefile.am
index 411ea0f..4ec61ce 100644
--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -31,7 +31,7 @@ systemdservicedir       = $(systemdsystemunitdir)
 systemdservice_DATA     = $(systemdservice_in_files:.service.in=.service)
 
 $(systemdservice_DATA): $(systemdservice_in_files) Makefile
-	@sed -e "s|\@libexecdir\@|$(prefix)/lib/udisks|" $< > $@
+	@sed -e "s|\@libexecdir\@|$(prefix)/lib/udisks|" $< > $(top_srcdir)/data/$@
 endif
 
 udevrulesdir = $(slashlibdir)/udev/rules.d
-- 
1.7.10.4

